.eleventy.js

```diff
@@ -0,0 +1,10 @@
+module.exports = function(eleventyConfig) {
+    return {
+        templateFormats: [
+            "njk",
+            "md",
+        ],
+        htmlTemplateEngine: "njk",
+        dataTemplateEngine: "njk",
+    }
+};
```

Comment:
This means that 11ty will create HTML pages from all Nunchuks (NJK) and Markdown (MD) files that exist in our project.